google-cloud-logging/lib/google/cloud/logging/resource_descriptor/list.rb

```diff
@@ -75,7 +75,7 @@ def next
             list_grpc = @service.list_resource_descriptors(
               token: token, max: @max
             )
-            self.class.from_grpc list_grpc, @service
+            self.class.from_grpc list_grpc, @service, @max
```

Comment:
Want to change the `max` param to keyword arg to match `Entry::List` and `Log::List`?